orangecanvas/application/addons.py

```diff
@@ -1382,7 +1382,7 @@ def uninstall(self, dist):
 
 class CondaInstaller:
     def __init__(self):
-        enabled = QSettings().value('add-ons/allow-conda-experimental',
+        enabled = QSettings().value('add-ons/allow-conda',
                                     False, type=bool)
```

Comment:
Should this be False here?

Reply:
It seems that this option has no effect since 'add-ons/allow-conda' key is always available. This is just a default option in case that key `add-ons/allow-conda` is not present. I changed it to true anyway.

Reply:
Ok, just so we don't miss something. :)

Reply:
@PrimozGodec, are you sure it is available even if a user did not open a settings dialog yet? Anyway, I think it would be better if we use the same default here. We should rely on other behavior as little as possible.

Reply:
@markotoplak I updated this setting in the last commit already.